pathd/path_pcep_cli.c

```diff
@@ -1186,11 +1189,192 @@ static void print_pcep_capabilities(char *buf, size_t buf_len,
 	}
 }
 
+/* Internal util function to print a pcep session */
+static void print_pcep_session_json(struct vty *vty, struct pce_opts *pce_opts,
+				    struct pcep_pcc_info *pcc_info,
+				    json_object *json)
+{
+	char buf[1024];
```

Comment:
Can we use something, but not static numbers? And initializing to zeros could be `buf[xxx] = {}`.

Reply:
BUFFER_PCE_SIZE